tests/test_pytest_plugin.py

```diff
@@ -32,7 +32,9 @@ def test_hello(test_client):
 
 @asyncio.coroutine
 def test_hello_with_loop(test_client, loop):
-    client = yield from test_client(create_app)
+    app = web.Application(loop=loop)
+    app.router.add_get('/', hello)
+    client = yield from test_client(app)
```

Comment:
You shouldn't modify existing tests as `test_client` must support both,
so please add separate tests for this feature.